spec/objects/group_spec.rb

```diff
@@ -44,7 +44,7 @@
     it 'builds images url' do
       expect(subject.urls(size: '200x200').first).
         to eq(
-          "http://www.ucarecdn.com/#{ subject.uuid }/nth/0/-/resize/200x200/"
```

Comment:
I don't think this solves the problem.
`urls` uses `api.options[:static_url_base]`, and the spec should use it too.